### PR TITLE
Clear documantion on the new release.

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

**Comment by samuelgarcia** [comment]
@alejoe91 : merge this if this is OK for you.

File: README.md [hunk 1]
```diff
@@ -63,6 +61,7 @@ SpikeInterface is a Python framework designed to unify preexisting spike sorting
 
 All documentation for SpikeInterface can be found [here](https://spikeinterface.readthedocs.io/en/latest/).
 
+Documentation of current release with OLD api is [here](https://spikeinterface.readthedocs.io/en/stable/).
```
Comment (alejoe91) [inline README.md:64]: @samuelgarcia what about CURRENT. I think that OLD gives the idea that people should not use it (but it's not the case yet)

File: doc/index.rst [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
 
+
+
+.. warning::
+
+    This is the not yet released documentation of spikeinterface (0.90.0)
+    
+    To see reelased documention this is here https://spikeinterface.readthedocs.io/en/stable
+    
+    Actual OLD API release is 0.12.0.
```
Comment (alejoe91) [inline doc/index.rst:12]: CURRENT

File: doc/installation.rst [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
-
-    pip install spikeinterface
-
-The pip installation will install a specific and fixed version of the spikeinterface packages.
+The actual "new API" (v0.90.0) is not release on pypi yet.
+It will release
+ in July 2021
 
-To use the latest updates, install `spikeinterface` and the related packages from source:
+You have to do this to install `spikeinterface` afrom source:
```
Comment (alejoe91) [inline doc/installation.rst:10]:
```suggestion
To use it now, you have to install `spikeinterface` from source:
```

**Review by alejoe91 — CHANGES_REQUESTED** [review]

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**